octobot_backtesting/importers/exchanges/exchange_importer.pxd

```diff
@@ -22,6 +22,7 @@ cdef class ExchangeDataImporter(DataImporter):
     cdef public list symbols
     cdef public list time_frames
 
+    cdef dict get_available_time_frames_by_symbol(self)
```

Comment:
It seems that this method is called by a python method in API.
I think you have to declare it with `cpdef` instead of `cdef`.